src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -42,14 +43,16 @@ static int Last2DigitsTo0(int versionBuild)
             _sdkRootPath = sdkRootPath;
             _sdkVersionBand = sdkVersionBand;
 
-            var manifestDirectoryEnvironmentVariable = Environment.GetEnvironmentVariable("DOTNETSDK_WORKLOAD_MANIFEST_ROOT");
+            var manifestDirectory = Path.Combine(_sdkRootPath, "sdk-manifests", _sdkVersionBand);
+
+            var manifestDirectoryEnvironmentVariable = Environment.GetEnvironmentVariable("DOTNETSDK_WORKLOAD_MANIFEST_ROOTS");
```

Comment:
I prefer not to change global value of the process. Even if we force it to run in parallel. It will also interfere any test without the test hook. And these test will still run slower when done locally.

The tricky part of the logic is what to do when there are multiple manifest directory instead of if Environment.GetEnvironmentVariable can get the value. So it is fine to have a test constructor of a test method to add an additional manifest directory.